<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinkowskiDimCalculatorDialogBase</class>
 <widget class="QDialog" name="MinkowskiDimCalculatorDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minkowski Dimension Calculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Layer (Linestring):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="onlySelectedFeaturesCheckBox">
     <property name="text">
      <string>Only for selected features</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="layerGridTab">
      <attribute name="title">
       <string>Layer grid</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="layerGridGroupBox">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QLineEdit" name="layerGridAttributeNameLineEdit">
            <property name="toolTip">
             <string>Name of field that will be created or recalculated for vector layer</string>
            </property>
            <property name="text">
             <string>mink_dem</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="layerGridNumberOfStepsSpinBox">
            <property name="toolTip">
             <string>This is how much grids will be calculated</string>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="layerGridAutoButton">
            <property name="text">
             <string>Auto</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="layerGridStartSizeLineEdit">
            <property name="toolTip">
             <string>Size of cell at first step. The biggest one. You can choose it based on extent - 4 or 5 cells in a row is good start</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="layerGridEndSizeLineEdit">
            <property name="toolTip">
             <string>Size of cell on last step. The smallest one. For maximal accurace it can be near smallest distance between feature's nodes. For reason of optimization can be larger, depending on scale of your research</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Attribute name</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Number of steps</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Start cell size</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>End cell size</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="featureGridTab">
      <attribute name="title">
       <string>Feature grid</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="featureGridGroupBox">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Attribute name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QLineEdit" name="featureGridAttributeNameLineEdit">
            <property name="toolTip">
             <string>Name of field that will be created or recalculated for vector layer</string>
            </property>
            <property name="text">
             <string>mink_dem</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Start cell size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="featureGridAutoCheckBox">
            <property name="text">
             <string>Auto</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>End cell size</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Number of steps</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QgsFieldComboBox" name="featureGridStartSizeComboBox"/>
          </item>
          <item row="2" column="2">
           <widget class="QgsFieldComboBox" name="featureGridEndSizeComboBox"/>
          </item>
          <item row="3" column="2">
           <widget class="QgsFieldComboBox" name="featureGridNumberOfStepsComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QgsMapLayerComboBox" name="layerComboBox"/>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Use &quot;Layer grid&quot; mode if features in your layer are comparable in size and MD could be calculated with common grid. Otherwise, use &quot;Feature grid&quot;. Auto-settings are OK for current layer objects comparison, but for real absolute values better to use manual settings.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="helpLink">
     <property name="text">
      <string>&lt;a href=&quot;http://www.ekazakov.info/projects/mdc_tutorial.html&quot;&gt;Help&lt;/a&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
